server/graphql/v2/object/VirtualCard.ts

```diff
@@ -48,7 +48,10 @@ export const VirtualCard = new GraphQLObjectType({
       type: GraphQLString,
       async resolve(virtualCard, _, req) {
         const collective = await req.loaders.Collective.byId.load(virtualCard.CollectiveId);
-        if (req.remoteUser?.isAdminOfCollectiveOrHost(collective) && checkScope(req, 'virtualCards')) {
+        if (
+          (req.remoteUser?.isAdminOfCollectiveOrHost(collective) || req.remoteUser?.id === virtualCard.UserId) &&
+          checkScope(req, 'virtualCards')
```

Comment:
Maybe add a small helper like `canSeeVirtualCardPrivateInfo` to reduce code duplication and prevent mistakes?

Reply:
Thanks. I've added the helper. 👍🏽